src/pages/[platform]/build-a-backend/server-side-rendering/index.mdx

```diff
@@ -33,4 +23,391 @@ export function getStaticProps(context) {
   };
 }
 
-<Overview childPageNodes={props.childPageNodes} />
+This guide walks through how to use Amplify Auth, GraphQL API, REST API, and Storage category APIs from Next.js server-side runtimes.
+
+import versionRangeCallout from '/src/fragments/lib/ssr/nextjs/version-range-callout.mdx';
+
+<Fragments fragments={{ js: versionRangeCallout }} />
+
+Before you begin, you will need:
+- [A Next.js application created](/[platform]/start/project-setup/create-application/)
+- [Installed and configured Amplify libraries for Next.js](/[platform]/build-a-backend/server-side-rendering/)
+
+## Configure Amplify Library for server-side usage
+
+To use Amplify APIs on the server-side of your Next.js app, you will need to
```

Comment:
Doesn't have to be fixed today but this page looks like they have hard breaks for each line which may impact page views on diff screens